docs/content/manual/release-specific/v1.7.0/test-rwx-fast-failover.md

```diff
@@ -0,0 +1,89 @@
+---
+title: RWX Fast Failover
+---
+
+## Related issues
+
+- https://github.com/longhorn/longhorn/issues/6205
+
+## LEP
+
+- https://github.com/longhorn/longhorn/pull/9069
+
+## Test step
+
+### Scale Test
+**Given** Longhorn cluster with 3 worker nodes.
+
+Make multiple deployments with a script such as
+```shell
+#!/bin/bash
+
+for i in {1..70}; do
+  cat <<EOF | kubectl apply -f -
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: rwx-volume-pvc-$i
+  namespace: default
+spec:
+  accessModes:
+    - ReadWriteMany
+  volumeMode: Filesystem
+  storageClassName: longhorn-test
+  resources:
+    requests:
+      storage: 200Mi
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: app-$i
+  namespace: default
+spec:
+  replicas: 3
+  selector:
+    matchLabels:
+      app: app-$i
+  template:
+    metadata:
+      labels:
+        app: app-$i
+    spec:
+      containers:
+      - name: app
+        image: busybox
+        command: ["/bin/sh", "-c", "exec 7<>/data/testfile-$(hostname -s); flock -x 7; while date | dd conv=fsync >&7 ; do sleep 1; done"]
+        volumeMounts:
+        - mountPath: /data
+          name: rwx-volume-pvc-$i
+      volumes:
+      - name: rwx-volume-pvc-$i
+        persistentVolumeClaim:
+          claimName: rwx-volume-pvc-$i
+EOF
+done
+```
+
+**Then** with the `rwx-enable-fast-failover` setting off, check the CPU and memory use totals for longhorn-manager and longhorn-share-manager pods, using something like `kubectl -n longhorn-system top pod`.  Establish baseline values.
+
+**Then** set `rwx-enable-fast-failover` to true, and scale down and up the deployments so that they will start updating and monitoring the leases.
+
+**Verify** that the CPU and memory use grows, but only by a small amount.
+
+**If possible** monitor the API server requests similar to the method in the report https://github.com/longhorn/longhorn/blob/master/scalability/reference-setup-performance-scalability-and-sizing-guidelines/public-cloud/medium-node-spec.md#longhorn-control-plane-performance
+
+**Verify** that the API request rate remains low.
+
+**Reference** How to set up a Grafana Testing Environment on Rancher
+https://github.com/longhorn/longhorn/issues/6205#issuecomment-2264430975
+
+### Failover Test
+**Given** Longhorn cluster with 3 worker nodes.
+
+**With** the same deployments as above (but perhaps only 20-30 of them), and fast failover enabled,
+    
+**Then** pick a node and restart it.  
+
+**Verify** the share-managers on that node being recreated one of the remaining nodes.  RWX volumes not on the failed node should continue to operate without any disruption.
```

Comment:
Do you mean, even when the share manager pods on the powered down node are recreated, the workload pods are not recreated and they will continue working?

Reply:
I should make this clearer. Volume A, whose share-manager pod was running on the failed node, is moved to a new node.  Its workload will be disrupted, of course.  What I mean is that if the share-manager pod for any volume B is not on the failed node, then that volume and its workload should continue on unaffected.